rpm/SPECS/openresty-openssl.spec

```diff
@@ -35,26 +39,22 @@ Provides C header and static library for OpenResty's OpenSSL library.
 
 %prep
 %setup -q -n openssl-%{version}
-
 %patch0 -p1
 
 
 %build
-./config --prefix=%{openssl_prefix} \
-    no-threads \
-    shared -g
-
+./config --openssldir=%{_prefix} no-threads shared zlib -g \
```

Comment:
@zxcvbn4038 OK, we can use `--openssldir`, but please point the DIR to the original `%{openssl_prefix}` location. I just wonder why you changed it to `%{_prefix}` in your branch.